app/assets/stylesheets/views/article.scss

```diff
@@ -332,6 +332,12 @@
   }
 }
 
+.crayons-post_sidebar_display_ad {
```

Comment:
Do you think we should maybe add this to the html since we have the media query classes in tailwind?

`s:`, `m:` and `l:`

Reply:
I hadn't thought of doing this. I tried it out and I saw two issues:

* Primarily: It _looks_ like `max-lg:hidden` or `max-lg:collapse` _ought_ to work here, but... I'm not sure it works? Possibly error is on my end, somehow, but also possibly...
* [tailwindcss seems to encourage us](https://tailwindcss.com/docs/responsive-design#working-mobile-first) to strategize "mobile-first" with general/unprefixed styles and use prefixes for _min-width_ more than _max-width_